package.json

```diff
@@ -15,10 +22,11 @@
     "markdown-it-inline-comments": "^1.0.1",
     "markdown-it-math": "^4.1.1",
     "markdown-it-strikethrough-alt": "^1.0.0",
-    "markdown-it-sub": "^2.0.0",
-    "markdown-it-sup": "^2.0.0",
```

Comment:
These were not used, the `-alt` packages have been used, they were just transitively imported.